jax/_src/distributed.py

```diff
@@ -26,6 +26,7 @@
 from jax._src.lib import xla_extension
 
 class State:
+  process_id: int = 0
```

Comment:
Shouldn't this be None as default to match the current behavior?

Reply:
It doesn't greatly matter. If you pass `None` to the underlying client init function, it defaults to 0. This is simple: always an int (after all, you have to have a node id, even if not distributed.)

Reply:
The current behavior is to error if process_id is None.

For TPUs we figure that thing out.